torch/package/package_exporter.py

```diff
@@ -493,6 +499,29 @@ def save_module(self, module_name: str, dependencies=True):
 
         self._intern_module(module_name, dependencies)
 
+    def _is_package(
+        self,
+        module_obj: types.ModuleType
+    ):
+        """Checks if module_obj is a package and throws an error, if module_obj is an
+            invalid package.
+        """
+        error = BadPackageError(f"{module_obj.__name__} is an improperly formed package "
```

Comment:
Can we improve the error message to explain why the package is improperly formed? Most of our users probably don't know anything about the Python import/module system.